pkg/operatormetrics/wrapper_registry_test.go

```diff
@@ -49,6 +49,30 @@ var _ = Describe("Registry", func() {
 		})
 	})
 
+	Describe("UnregisterMetrics", func() {
+		BeforeEach(func() {
+			err := CleanRegistry()
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		It("should unregister metrics without error", func() {
+			counter := NewCounter(testCounterOpts)
+			gauge := NewGauge(testGaugeOpts)
+
+			err := RegisterMetrics([]Metric{counter, gauge})
+			Expect(err).NotTo(HaveOccurred())
+
+			Expect(operatorRegistry.registeredMetrics).To(HaveLen(2))
+			Expect(operatorRegistry.registeredMetrics).To(HaveKey(testCounterOpts.Name))
+			Expect(operatorRegistry.registeredMetrics).To(HaveKey(testGaugeOpts.Name))
+
+			err = UnregisterMetrics([]Metric{counter, gauge})
```

Comment:
nit: can you add an additional metric that should not be removed?

Reply:
This is really important, thanks!
Done.